requests-responses.md

```diff
@@ -212,12 +217,14 @@ The signature must be verified using the public key provided via `get_public_key
 interface AppRequest {
 	method: string;
 	params: string[];
+    return: 'back' | 'none' | '<my-return-url>';
 	id: string;
 }
 ```
 Where 
 - method: name of the operation ('sendTransaction', 'singMessage', ...)
 - params: array of the operation specific parameters
+- return: return strategy for deeplinks when user signs/declines the request. 'back' means return to the app which initialized deeplink jump (e.g. browser, native app, ...), 'none' means no jumps after user action. If a specific url is passed, wallet will open this url deeplink after user's action. Note, that you shouldn't pass your app url if it is a webpage. This option should be used for native apps to prevent 'back' strategy OS bugs.
```

Comment:
Please format the list of options nicely, as mentioned above. Maybe even have a separate section for "Return options" and cross-ref it from both points in the spec.

Reply:
done